### PR TITLE
admin create user with role

### PR DESCRIPTION
**Pull request by hienmvvn**

### CODE REVIEW

File: app/controllers/admin/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Admin::UsersController < ApplicationController
+  load_and_authorize_resource
+  
+  def index
+    @users = User.all
+  end
+
+  def create
+    @user = User.new user_params
```
Comment (tanphan313) [inline app/controllers/admin/users_controller.rb]: remove

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class Admin::UsersController < ApplicationController
+  load_and_authorize_resource
+  
+  def index
+    @users = User.all
```
Comment (tanphan313) [inline app/controllers/admin/users_controller.rb]: remove

File: app/views/admin/courses/index.html.erb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 <% if can? :manage, :all %>
   <%= link_to t("course.create"), new_admin_course_path, class: "btn btn-primary" %>
   <%= link_to t("subject.all_subject"), admin_subjects_path, class: "btn btn-primary" %>
+  <%= link_to t("user.all_user"), admin_users_path, class: "btn btn-primary" %>
+
```
Comment (ngocquyhoang) [inline app/views/admin/courses/index.html.erb]: remove this line

Comment (tanphan313) [inline app/views/admin/courses/index.html.erb]: góe á jíu náo???

**Comment by tanphan313** [comment]
ok

**Comment by ngocquyhoang** [comment]
OK

**Pull request merged**